opae-libs/cmake/modules/FindCap.cmake

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/cmake -P
```

Comment:
This file is being added to opae-libs, so it must be added at https://github.com/OPAE/opae-libs.git.

Reply:
Done the changes and pushed to opae-libs.git